kgraphql-ktor/src/test/kotlin/com/apurebase/kgraphql/FieldsTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.apurebase.kgraphql
+
+import io.ktor.util.KtorExperimentalAPI
+import kotlinx.serialization.UnstableDefault
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+
+class FieldsTest {
+    data class Fields(val stuff: String, val moreStuff: String)
+
+    @UnstableDefault
+    @KtorExperimentalAPI
+    @Test
+    fun `All fields can be read is case of a single entity`() {
+        val server = withServer {
+            query("actor") {
+                resolver { ctx: Context ->
+                    val fields = ctx.fields().joinToString(",")
```

Comment:
Probably there's a better way to access this that to echoing them in the resolver